.meta/sinks/papertrail.toml

```diff
@@ -0,0 +1,34 @@
+[sinks.papertrail]
+title = "Papertrail"
+beta = true
+common = false
+delivery_guarantee = "best_effort"
+egress_method = "streaming"
+function_category = "transmit"
+healthcheck = true
+input_types = ["log"]
+write_to_description = "[Papertrail](https://www.papertrail.com/)"
+
+<%= render("_partials/_component_options.toml", type: "sink", name: "papertrail") %>
+
+<%= render("_partials/_buffer_options.toml", namespace: "sinks.papertrail.options") %>
+
+[sinks.papertrail.options.endpoint]
+type = "string"
+common = true
+required = true
+examples = ["logs.papertrailapp.com:12345"]
```

Comment:
Does this have a default?

Reply:
Nope, they use the host and port as the "token" in a sense. So you must always provide this.